core/PRP/Camera/plCameraModifier.cpp

```diff
@@ -155,6 +155,15 @@ void plCameraModifier::write(hsStream* S, plResManager* mgr) {
     S->writeBool(fResetAnimOnPop);
 }
 
+bool plCameraModifier::orderAfter(const hsKeyedObject* other) const {
+    // No strong association between plCameraModifier and plAGMasterMod,
+    // so just ensure this is always sorted after all plAGMasterMods
+    // TODO: Improve this if possible
+    if (other->getKey()->getType() == kAGMasterMod)
```

Comment:
AGMasterMod is indeed the direct dependency (in some situations). However, CameraModifier absolutely must be after the AGModifier as well

Reply:
Since the AGMasterMod will cause all AGModifiers to be before their associated AGMasterMods, sorting the CameraModifier after all AGMasterMods will automatically ensure that CameraModifier is also after all AGModifiers.